### PR TITLE
Ucrscript

### PR DESCRIPTION
**Pull request by maxwellfundi**

### CODE REVIEW

**Comment by maxwellfundi** [comment]
@dannyparsons  this now fixes your issue of shortcuts but i have one more improvement to suggest
When pasting, with no selected text, I suggest we paste where we have the current cursor position not at the end of the text in the text box. I cant remember why we did it that way. 
What do you think?

**Comment by dannyparsons** [comment]
Yes always paste at the cursor position. This should all work as Word does for example.

**Comment by maxwellfundi** [comment]
@dannyparsons  thats now fixed. You can test this now.

File: instat/ucrScript.vb [hunk 1]
```diff
@@ -130,34 +130,20 @@ Public Class ucrScript
     End Sub
 
     Private Sub mnuCopy_Click(sender As Object, e As EventArgs) Handles mnuCopy.Click
-        Clipboard.SetText(txtScript.Text)
+        Clipboard.SetText(txtScript.SelectedText)
```
Comment (dannyparsons) [inline instat/ucrScript.vb]: Why arent' you using textbox.Copy and textbox.Cut?

Comment (maxwellfundi) [reply instat/ucrScript.vb]: I could use that too, they work in same way.

Comment (dannyparsons) [reply instat/ucrScript.vb]: We should be consistent. In another place you are using textbox.Paste. I suggest we follow the documentation https://msdn.microsoft.com/en-us/library/system.windows.forms.textboxbase.copy(v=vs.110).aspx

Comment (maxwellfundi) [reply instat/ucrScript.vb]: Okay.

**Pull request merged**